Packs/CloakedUrsaPhishingCampaign/Playbooks/playbook-Cloaked_Ursa_Diplomatic_Phishing_Campaign.yml

```diff
@@ -0,0 +1,1884 @@
+          - operator: uniq
+      QRadarTimeFrame:
+        complex:
+          root: inputs.QRadarTimeRange
+      SHA256:
+        complex:
+          root: File
+          accessor: SHA256
+          transformers:
+          - operator: uniq
+      SplunkEarliestTime:
+        complex:
+          root: inputs.SplunkEarliestTime
+      SplunkLatestTime:
+        simple: now
+      URLDomain:
+        complex:
+          root: Domain
+          accessor: Name
+          transformers:
+          - operator: uniq
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": -780,
+          "y": 1810
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "19":
+    id: "19"
+    taskid: ae82f63b-6fc0-464b-8ff8-e6572c942d1b
+    type: regular
+    task:
+      id: ae82f63b-6fc0-464b-8ff8-e6572c942d1b
+      version: -1
+      name: Tag Email Indicators
+      description: Create indicators to the Threat Intel database only if they are not registered. When using the script with many indicators, or when the Threat Intel Management database is highly populated, this script may have low performance issue.
+      scriptName: CreateNewIndicatorsOnly
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "8"
+    scriptarguments:
+      indicator_values:
+        simple: dawid.tomaszewski@resetlocations.com, ops.rejon4@kazmierz.pl
+      source:
+        complex:
+          root: http.parsedBlog
+          accessor: sourceLink
+      tags:
+        simple: APT29, UAC-0029, Midnight Blizzard, Nobelium, Cozy Bear
+      type:
+        simple: Email
+      verdict:
+        simple: Malicious
+    reputationcalc: 2
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1070,
+          "y": 900
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "20":
+    id: "20"
+    taskid: 38fa6943-96ce-490e-81b0-379418bae5a7
+    type: regular
+    task:
+      id: 38fa6943-96ce-490e-81b0-379418bae5a7
+      version: -1
+      name: Tag URL Indicators
+      description: Create indicators to the Threat Intel database only if they are not registered. When using the script with many indicators, or when the Threat Intel Management database is highly populated, this script may have low performance issue.
+      scriptName: CreateNewIndicatorsOnly
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "8"
+    scriptarguments:
+      indicator_values:
+        complex:
+          root: URL
```

Comment:
There is no chance that the Blog URL will be here as well right? (due to the `collect IOCS` task)

Reply:
There is always a chance but it shouldn't